### DISCUSSION TITLE
Limit on requests per period + limits on query length

### DISCUSSION THREAD

**Discussion by DerpLady** [OP-TED/eForms-SDK#1059] [Q&A]
Hi team! 
I cant find the answer to my issue in the API documentation for the notice search with scroll in TED: https://ted.europa.eu/api/documentation/index.html#/notice-search-v3/search

I get 504 when I recently increased the search query string length and a HTTP Error 429 (Too many requests) when retrieving XML files.

1. What is the formal query length limit? The notice limit info doesn't make sense to me (is it 10 000, 250?)
2. How many queries to the API is allowed per time range (e.g. 10 per day, per hour, other?)

**Comment by rouschr** [comment]
You may occasionally get TED API related answers here especially if they are related to the SDK (rules/validation), but this seems to be TED API specific so you contact the TED helpdesk: https://ted.europa.eu/en/contact
